src/doc/rustdoc/src/linking-to-items-by-name.md

```diff
@@ -62,4 +74,17 @@ struct Foo {}
 fn Foo() {}
 ```
 
-Note: Because of how `macro_rules` macros are scoped in Rust, the intra-doc links of a `macro_rules` macro will be resolved relative to the crate root, as opposed to the module it is defined in.
+You can also disambiguate for functions by adding `()` after the function name,
```

Comment:
Didn't you already say this above? https://github.com/rust-lang/rust/pull/76934/files/39897bc1b740fdd849d506f28a4185823bfc5a02..51b99bed5be71a78bf0f29636cbcc905e4658f35#diff-889559de5f336fddc5bc898caab5a4e5R31

Reply:
Yes, but it doesn't explicitly say that this works for disambiguation.